UIKit_DZ_Swift2.0/UIKit_DZ_Swift2.0/ViewControllers/CheckViewController.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  CheckViewController.swift
+//  UIKit_DZ_Swift2.0
+//
+//  Created by Ernest Avagovich on 28.09.2022.
+//
+
+import UIKit
+
+/// контроллер отображающий итоговый заказ и способы оплаты данного заказа
+class CheckViewController: UIViewController {
+
+    // MARK: - Public properties
+    public var pizza: Pizza?
+    public var ingrediants: [String?] = []
+    public var goToBackClouser: (() -> ())?
+    public weak var delegate: PopToFoodViewControllerDelegate?
```

Comment:
```suggestion
    weak var delegate: PopToFoodViewControllerDelegate?
```
public - никогда не используем